webpack.config.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index :'./src/index.js'
+},
+  devServer: {
+    static: './dist',
+  },
+
+  plugins: [
+    new HtmlWebpackPlugin({
+        // title: 'Output Management',
+        template: './src/index.html',
+    }),
+  ],
+
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+        // the code below must be activate when we have more than one js file
+//   optimization: {
+//     runtimeChunk: 'single',
+//   },
```

Comment:
- [ ] Kindly remove this commented CSS code

Reply:
done